src/Items/ItemChest.h

```diff
@@ -28,13 +28,15 @@ class cItemChestHandler:
 		int a_CursorX, int a_CursorY, int a_CursorZ
 	) override
 	{
+		Vector3i absPos(a_BlockX, a_BlockY, a_BlockZ);
```

Comment:
butWhy

And what was wrong with keeping blockPos down there? Seems like using a_BlockY in the valid height check makes for a shorter line.

Reply:
Preparation for the conversion to Vector3s everywhere! We need vector3s in one place so we should use them everywhere for consistency. And if they're everywhere then the defintion needs to be at the start of the function.

Reply:
Would you consider `BlockPos` instead?

Reply:
This is due to be refactored again, that is indeed a better name.